docs/integrations/builtin/credentials/couchbase.md

```diff
@@ -0,0 +1,40 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Couchbase credentials
+description: Documentation for Couchbase credentials. Use these credentials to authenticate Couchbase in n8n, a workflow automation platform.
+contentType: integration
+priority: medium
+---
+
+# Couchbase credentials
+
+You can use these credentials to authenticate the following nodes:
+
+- [Couchbase](/integrations/builtin/app-nodes/n8n-nodes-base.couchbase/)
+
+## Prerequisites
+
+- Create a user account with the appropriate permissions on a [Couchbase](https://www.couchbase.com/){:target=_blank .external-link} server.
+
+If you are setting up Couchbase from scratch, create a cluster and a bucket. Refer to the [Couchbase documentation](https://docs.couchbase.com/home/index.html){:target=_blank .external-link} for more detailed instructions on these steps.
+
+## Supported authentication methods
+
+- Database connection - Values
```

Comment:
I think we're good to just say "Database connection", since we only support one method of DB connection (ref for my thinking [here](https://docs.n8n.io/integrations/builtin/credentials/mysql/))
 ```suggestion
- Database connection
```